src/qt/CMakeLists.txt

```diff
@@ -15,19 +14,13 @@ set(APP_SOURCES
 )
 
 set(RESOURCES
-   audacity.qrc
```

Comment:
I think no, as I understand *.qml should be added either to qrc or via `qt_add_qml_module` not both. As I explained in the commit comment I reverted it because right now this is the only source for deploy script where it can scan for imported qt qml modules